mp4/meta.go

```diff
@@ -34,44 +47,44 @@ func (b *MetaBox) AddChild(child Box) {
 	b.Children = append(b.Children, child)
 }
 
-// DecodeMeta - box-specific decode
+// DecodeMeta decodes a MetaBox in either MPEG or QuickTime version.
```

Comment:
you might prefer to remove the `"."` here and below

Reply:
yup. fixing.